docs/docs/ip-per-task.md

```diff
@@ -6,20 +6,15 @@ title: IP-per-task
 
 <div class="alert alert-danger" role="alert">
   <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span> Available in Marathon Version 0.14.0+ <br/>
-    The IP-per-task functionality is considered experimental, so use this
-    feature at your own risk. We might add, change, or delete any
-    functionality described in this document, including the API. We
-    appreciate <a href="https://github.com/mesosphere/marathon/issues/2709">your feedback</a> !
+    IP-per-task functionality is experimental, so use this feature at your own risk. We might add, change, or delete any functionality described in this document, including the API. We appreciate <a href="https://github.com/mesosphere/marathon/issues/2709">your feedback</a> !
 </div>
 
 
-In version 0.14, Marathon introduced experimental IP-per-task support. With the appropriate configuration,
-every tasks of an app gets its own network interface and IP address.
+In version 0.14, Marathon introduced experimental IP-per-task support. With the appropriate configuration, every tasks of an app gets its own network interface and IP address.
 
-This can drastically simplify service discovery, since you can use
-[mesos-dns](https://github.com/mesosphere/mesos-dns) to rely on DNS for service discovery. This enables your
-applications to use address-only (A) DNS records in combination with known ports to connect to other
-services -- as you would do it in a traditional static cluster environment.
+This drastically simplifies service discovery, since you can use
+[Mesos-DNS](https://github.com/mesosphere/mesos-dns). Mesos-DNS enables your
+applications to use address-only (A) DNS records in combination with known ports to connect to other services, as you would do it in a traditional static cluster environment.
 
 You can request an IP-per-task with default settings like this:
```

Comment:
The example shows an app definition. Since you raise the question, it is not clear. Please add this.